eq-author-api/package.json

```diff
@@ -53,7 +54,8 @@
       "!scripts/*",
       "!middleware/import.js",
       "!app.js",
-      "!utils/datastoreFileSystem.js"
+      "!utils/datastoreFileSystem.js",
+      "!utils/migrations/*"
```

Comment:
I would say we should cover this

Reply:
Once I move this to `/scripts`, it won't be covered due the existing ignore rule.  We don't test other scripts, why would we test this?

Reply:
Yeah, sorry I missed that migrations end up as `/migrations` and not `utils/migrations` so I wanted to make sure those were covered.